client/reader/following-manage/feed-search-results.jsx

```diff
@@ -1,115 +1,115 @@
 /**
  * External Dependencies
  */
-import React from 'react';
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
-import { take, map, times } from 'lodash';
+import { take, times } from 'lodash';
 import Gridicon from 'gridicons';
 import classnames from 'classnames';
 
 /**
  * Internal Dependencies
  */
-import ConnectedSubscriptionListItem from 'blocks/reader-subscription-list-item/connected';
-import SitesWindowScroller from './sites-window-scroller';
 import Button from 'components/button';
 import ReaderSubscriptionListItemPlaceholder
 	from 'blocks/reader-subscription-list-item/placeholder';
 import { READER_FOLLOWING_MANAGE_SEARCH_RESULT } from 'reader/follow-button/follow-sources';
-import { recordTracksRailcarRender } from 'reader/stats';
+import InfiniteStream from 'components/reader-infinite-stream';
+import { siteRowRenderer } from 'components/reader-infinite-stream/row-renderers';
+import { getReaderFeedsCountForQuery } from 'state/selectors';
+import { requestFeedSearch } from 'state/reader/feed-searches/actions';
 
-const FollowingManageSearchFeedsResults = ( {
-	showMoreResults,
-	showMoreResultsClicked,
-	searchResults,
-	translate,
-	width,
-	fetchNextPage,
-	forceRefresh,
-	searchResultsCount,
-	query,
-} ) => {
-	const isEmpty = !! ( query && query.length > 0 && searchResults && searchResults.length === 0 );
-	const classNames = classnames( 'following-manage__search-results', {
-		'is-empty': isEmpty,
-	} );
+class FollowingManageSearchFeedsResults extends React.Component {
```

Comment:
Everything is good about it. Was able to consolidate a couple of logic branches.  It just shouldn't have been intermixed with this PR that is focused on swapping out SitesWindowScroller --> infinite-stream